tests/integration/control/test_control_attacks.py

```
@@ -0,0 +1,983 @@
+from app.models.project import ProjectUserAssociation, ProjectUserRole
+from app.models.user import User
+from tests.factories.attack_factory import AttackFactory
+from tests.factories.campaign_factory import CampaignFactory
+from tests.factories.hash_list_factory import HashListFactory
+from tests.factories.project_factory import ProjectFactory
+
+
+async def create_wordlist_resource(
+    db_session: AsyncSession,
+    project_id: int | None = None,
+    is_uploaded: bool = True,
```

Ruff (0.15.0)

[error] Boolean-typed positional argument in function definition

(FBT001)